src/MentionableTextarea.vue

```diff
@@ -245,6 +247,16 @@
                     this.identifierControl();
                 }
             },
+            scrollSelectedIntoView() {                
+                let parentUL = document.getElementById(this.elementIdMap.get('UL'));                
```

Comment:
I'm glad it works!

Reply:
> > I still can't find what is the bug here sweat_smile Looked at it like 15 times till now.
> > 
> > 1. What should be expected?
> > 2. What is happening?
> > 
> > It'll be better if we open a new issue, if it is infact an issue.
> 
> It's the mention box's position. If you look closely to the last frame, you can see that the suggestion box opens in the **right** side of the textbox, even though I'm only starting to write from the **left** side.
> 
> We better open a new issue for it, you're right!

Oh. Ok. Now I see it. The position of the caret is not getting updated it seems. A new issue will be good.